app/javascript/menu/main-menu.jsx

```diff
@@ -172,7 +186,8 @@ export const MainMenu = ({
         <>
           <SideNav aria-label={__('Secondary Menu')} className="secondary" isChildOfHeader={false} expanded>
             <div onKeyDown={hideSecondaryEscape} role="presentation">
-              <SecondLevel menu={activeSection.items} hideSecondary={hideSecondary} />
+              <SecondLevel menu={activeSection.items} hideSecondary={hideSecondary} ref={secondLevelFirst} />
+              <span onFocus={hideSecondary} role="presentation" tabIndex="0" />
```

Comment:
Even the linter is complaining about this line and I don't like it either :confused: cc @himdel